docs/2.0/docs/patcher/concepts/grouping.md

```diff
@@ -0,0 +1,76 @@
+
+In summary, there are five ways to group pull requests generated by Patcher:
+1. **Full-Consolidation**: One PR to update everything.
+2. **No-Consolidation**: One PR for each update.
+3. **Dependency-Only Consolidation**: One PR per dependency. This allows you to merge one dependency at a time to all environments where they are used.
+4. **Environment-Only Consolidation**: One PR per environment. This allows you to promote all dependencies as a group through environments one at a time.
+5. **(Environment x Dependency) Consolidation**: One per dependency per environment. This allows you to promote a single dependency at a time through environments
+
+## Grouping Examples
+To clarify via example, let's imagine we have the following repository
+```
+/dev/unit1/terragrunt.hcl -> dependency1@1.0.0
+/dev/unit2/terragrunt.hcl -> dependency1@1.0.0
+/dev/unit3/terragrunt.hcl -> dependency2@1.0.0
+/prod/unit1/terragrunt.hcl -> dependency1@1.0.0
+/prod/unit2/terragrunt.hcl -> dependency1@1.0.0
+/prod/unit3/terragrunt.hcl -> dependency2@1.0.0
+/prod/unit4/terragrunt.hcl -> dependency3@1.0.0
+```
+
+In this example, assuming all 3 dependencies have newer versions available, the 5 strategies would result in:
+1. **Full-Consolidation**: One pull request with an update to all 7 units
+2. **No-Consolidation**: Seven pull requests, one for each unit
+3. **Dependency-Only Consolidation**: Three pull requests, one for each dependency, with the PRs for `dependency1` and `dependency2` updating both `dev` and `prod`.
+4. **Environment-Only Consolidation**: Two pull requests, one for `dev` and one for `prod`.
+5. **(Environment x Dependency) Consolidation**: Five pull requests - two for `dev` and three for `prod`.
+
+## Terminology
+* `unit` A unit refers to a folder containing a `terragrunt.hcl` file, and thus a single corresponding OpenTofu state file.  A unit may specify one or multiple modules as `dependencies`.
+* `dependency` (or `target`) A dependency is an OpenTofu module that is referenced by `ref` (usually a full source code path AND a version number) inside your `unit`.  Patcher understands the semantics of semantic versioning on dependency `ref`s.
+* `environment` is a logical grouping of infrastructure to represent your application environments, such as `dev` or `prod`.  An environment usually contains multiple `units` and thus many `dependencies`.  Generally IaC environments are similar to each other, and represented as a folder structure in your repository.
+* `update` is the the act of changing a single instance of a dependency to reference a newer version.
+    :::info
```

Comment:
did this `:::info` block get rendered correctly in the vercel app? it's not rendering in the markdown rich diff here in github, and i don't have access to vercel so i can't verify myself 😬

Reply:
i looked (thank you for access!!), this _does_ render correctly. apologies for the noise!